src/index.js

```diff
@@ -22,6 +23,14 @@ export class AddonStore {
     this._preview = preview;
   }
 
+  getDatabase() {
+    return this._database;
+  }
+
+  setDatabase(database) {
```

Comment:
That's upto them. This is a datastore provide by storybook. So, storybook should provide that, not an addon.

Let's addon comments addon uses a database used by us. Then suddenly another addon changes the database to something else. So, now comments save data in some other place.

So, we should be causes about this.